### PR TITLE
Readme.md updated

### PR DESCRIPTION
**Pull request by KristinaMatuleviciute**
message part added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,58 @@
...
+git clone https://github.com/rjrodger/nodezoo-info.git
+```
+
+Alternatively, fork the repository so that you can make your own changes.
+
+
+## Messages
+
+This micro-service recognizes the following messages:
+
```
Comment (mcdonnelldean) [inline README.md]: These are the messages for npm, you need to figure out the ones for this service and put them here.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,58 @@
 ![Nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
 
 # nodezoo-info
-nodezoo.com micro-service handling module. 
+nodezoo.com micro-service handling module.
+One of the micro-services that implement
```
Comment (mcdonnelldean) [inline README.md]: Line 5 - 24 remove

Comment (KristinaMatuleviciute) [inline README.md]: yes sorry saw it just now that its was wrong :)

Comment (mcdonnelldean) [inline README.md]: Not to worry :D

**Pull request closed without merging**